prow/plugins/config.go

```diff
@@ -184,6 +185,12 @@ type RequireSIG struct {
 	GroupListURL string `json:"group_list_url,omitempty"`
 }
 
+// Retitle specifies configuration for the retitle plugin.
+type Retitle struct {
+	// AllowClosedIssues allows retitling closed/merged issues and PRs.
+	AllowClosedIssues bool `json:"allow_closed_issues,omitempty"`
```

Comment:
Added a config option but words are failing me right now. I think we can come up with a better name...

Reply:
I think the name is fine